docs/manual/src/docs/asciidoc/_includes/reactive/oauth2/access-token.adoc

```diff
@@ -33,3 +33,46 @@ SecurityWebFilterChain configure(ServerHttpSecurity http) throws Exception {
 ----
 
 You can now leverage Spring Security's <<webclient>> or <<webflux-roac,@RegisteredOAuth2AuthorizedClient>> support to obtain and use the access token.
+
+== BearerTokenResolver
+
+With interface BearerTokenResolver you can provide a strategy to resolve a bearer token.
```

Comment:
When referring to class names, and other inlined code, please use backticks, e.g. `BearerTokenResolver`